machines/vcItem.ts

```diff
@@ -407,7 +407,11 @@ export const vcItemMachine =
               entry: 'removeVcItem',
               on: {
                 STORE_RESPONSE: {
-                  actions: ['removedVc', log('removing Vc')],
+                  actions: [
+                    'removedVc',
+                    log('removing Vc'),
```

Comment:
can we remove the loggers that is not needed.

Reply:
OK @vijay151096